docs/reference/openapi-rules.md

```diff
@@ -1,3 +1,546 @@
-## OpenAPI Rules
+# OpenAPI Rules
 
-Coming soon...
\ No newline at end of file
+Spectral has three rulesets built in: 
+
+- [oas](#oas)
+- [oas2](#oas2)
+- [oas3](#oas3)
+
+_OAS is shorthand for OpenAPI Specification._
+
+In your ruleset file you can add `extends: "spectral:oas"` and you'll get everything from `spectral:oas2` and `spectral:oas3`, meaning you can lint either type of document and only the relevant rules will apply thanks to [formats](../getting-started/rulesets.md#formats).
+
+Let's look at the rules in these rulesets.
+
+## oas
+
+### operation-2xx-response
+
+Operation must have at least one `2xx` response. Any API operation (endpoint) can fail but presumably it is also meant to do something constructive at some point. If you forget to write out a success case for this API, then this rule will let you know.
```

Comment:
Wow, just started reading this but I can see how much work you've done here! Curious if someone in the team knows a way how to automate this in the future (e.g. maybe we could read rules directly from the code and "compile" a doc)

Reply:
Code would have helped with formatting but so did copy and paste. The time killer was making examples and writing the human bits. If we automate from tests we get crap examples, and we would need to have written the human bits anyhow. 

At some point we'll want a tool for other people to use which can help them generate docs for their rulesets, but we already built an API Ref Doc tool and building a Ruleset Ref Doc tool right now seemed a bit much. :)